### ISSUE TITLE
Unable to find javacpp-1.5.10-android-arm.jar

### ISSUE THREAD

**Issue by junA2Z** [bytedeco/javacpp#768]
Hi team, my application is depending on `org.bytedeco:javacpp:1.5.10`. For some unknown reason, during the build, it's throwing error of
```
> Could not find javacpp-1.5.10-android-arm.jar (org.bytedeco:javacpp:1.5.10).
```
which indicates that it's trying to look for that package, and we can't find that package listed here: https://repo1.maven.org/maven2/org/bytedeco/javacpp/1.5.10/. I'm sure my application has nothing to do with android. We are using gradle. 
1. What is the potential reason for it? Why it consumes the android-arm package?
2. I'm seeing that [android-arm.properties](https://github.com/bytedeco/javacpp/blob/a5068201381120b0a8ef690f1068beb59d74c9c1/src/main/resources/org/bytedeco/javacpp/properties/android-arm.properties#L4) exists in this repo. Why didn't we publish that to 1.5.10, while it exists in 1.5.9?
3. Is there any approach to disable it to look for android-arm? 

Thanks!

**Comment by junA2Z** [comment]
I think it would be the most straightforward if someone could publish a `javacpp-1.5.10-android-arm.jar` to version 1.5.10, to unblock our project. Thanks!

**Comment by saudet** [comment]
Sure, contributions are welcome! Please start by getting an OSSRH account:
https://central.sonatype.org/publish/publish-guide/

**Comment by saudet** [comment]
Duplicate of https://github.com/bytedeco/javacpp-presets/issues/1483

**Comment by junA2Z** [comment]
Sorry to reopen. Is it intentionally excluded/unpublished, or it's not? Apparently you are more familiar with this repo than I'm, so without knowing why it wasn't published, or the root cause it was unpublished unexpectedly if unintentionally, I'm afraid the 32-bit will still remain unpublished, even if I spent amount of time on this.

**Comment by junA2Z** [comment]
And is it unclear to you as well that why we are consuming the android one, though I'm not compiling on android? Is it possible to exclude it during the build?

**Comment by saudet** [comment]
I see, you don't need it for your build, so don't include it in your dependencies?

**Comment by junA2Z** [comment]
I don't need android one for my build. We only specified `org.bytedeco:javacpp` in gradle,  but it's trying to automatically include the android 32-bit jar one, which is a mysterious part for me. I was expecting other users have similar issue.

**Comment by saudet** [comment]
Check the dependency tree of your project to see where it's getting included
